eva-accession-pipeline/automation/run_accessioning_in_embassy_cloud.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+
+# Copyright 2019 EMBL - European Bioinformatics Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import click
+import glob
+import json
+import subprocess
+import os
+import configparser
+import sys
+from __init__ import logger
+
+sys.path.append("../../eva-accession-import-automation")
```

Comment:
I guess there isn't a cleaner way to do this, right?

Reply:
We can't use relative path imports when module name has hyphens.